src/Liquid/Liquid.php

```diff
@@ -52,11 +52,14 @@ class Liquid
 		// Prefix for include files.
 		'INCLUDE_PREFIX' => '_',
 
+		// Whitespace control.
+		'WHITESPACE_CONTROL' => '-',
+
 		// Tag start.
-		'TAG_START' => '(?:{%|\s*{%-)',
+		'TAG_START' => '{%',
 
 		// Tag end.
-		'TAG_END' => '(?:%}|-%}\s*)',
+		'TAG_END' => '%}',
 
 		// Variable start.
 		'VARIABLE_START' => '{{',
```

Comment:
`VARIABLE_START` could have used the very same approach used for `TAG_START`. Why do you think this approach is inferior?

Reply:
It's not so much about inferiority as it is compatibility/simplicity. Having the start/end tags include the whitespace control:

1. Increases the complexity of customizing the start/end tags. Especially for those not familiar with regular expressions.
2. Means anyone who's already customized their start/end tags won't be able to use whitespace control unless they update them to include the new whitespace regex.

Reply:
Makes sense, thank you.